obal/data/modules/srpm.py

```diff
@@ -134,6 +136,10 @@ def main():
         if scl:
             command += ['--define', 'scl %s' %  scl]
 
+        if macros is not None:
```

Comment:
If the default is `{}`, is this ever None? I kind of forgot what `argument_spec` really does.

Reply:
If the `default` is `{}` then this is never `None`, no. It's only `None` for fields that don't have a default and are `required=False`.